src/main/kotlin/model/Dealer.kt

```diff
@@ -1,15 +1,18 @@
 package model
 
 class Dealer() : BasePlayer("Dealer") {
```

Comment:
```suggestion
class Dealer : BasePlayer("Dealer") {
```

`()` seems redundant here 👀

Reply:
Removed empty brackets